libgnucash/engine/gnc-commodity.cpp

```diff
@@ -167,100 +167,100 @@ struct gnc_quote_source_s
  * Apply changes here also to the FQ appendix of help.
  */
 static gnc_quote_source currency_quote_source =
-{ TRUE, 0, 0, "Currency", "CURRENCY", "currency" };
+{ TRUE, SOURCE_SINGLE, 0, "Currency", "CURRENCY", "currency" };
```

Comment:
This one should immediately be set to SOURCE_CURRENCY rather than setting it in the init routine further down.